src/osemgrep/cli_ci/Ci_subcommand.ml

```diff
@@ -703,7 +703,7 @@ let run_conf (ci_conf : Ci_CLI.conf) : Exit_code.t =
     *)
     let profiler = Profiler.make () in
     let targets_and_ignored =
-      Find_targets.get_targets conf.targeting_conf conf.target_roots
+      Find_targets.get_target_fpaths conf.targeting_conf conf.target_roots
```

Comment:
Why the _fpaths suffix? We are using more consistently fpaths now in semgrep, so we should assume
target filenames have Fpath.t type.
Or is it because we also define some Input_to_core.target and xtarget type?

Reply:
No, it's because the default `get_targets` function returns fppaths.

Reply:
I thought fppaths could be useful for troubleshooting or for further filtering. Maybe it's confusing?